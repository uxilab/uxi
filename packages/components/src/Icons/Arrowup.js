import React from 'react';
import PropTypes from 'prop-types';
import SvgIcon from '../SvgIcon';
/* eslint-disable max-len */

const Arrowup = props => (
  <SvgIcon {...props}>
    <svg
      focusable="false"
      xmlns="http://www.w3.org/2000/svg"
      viewBox="0 0 16 9.3"
      width="24px"
      height="24px"
    >
      <g id="Layer_2" data-name="Layer 2">
        <g id="Layer_1-2" data-name="Layer 1">
          <polyline points="8 0 0 8 1.3 9.3 8 2.6 14.7 9.3 16 8" />
        </g>
      </g>
    </svg>
  </SvgIcon>
);

Arrowup.propTypes = {
  color: PropTypes.string,
};

Arrowup.defaultProps = {
  color: '#6d6d71',
};

export default Arrowup;
