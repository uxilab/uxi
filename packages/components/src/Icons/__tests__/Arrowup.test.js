import React from 'react';
import Arrowup from '../Arrowup';

describe('Icon : <Arrowup />', () => {
  it('match snapshot', () => {
    expect(shallow(<Arrowup />)).toMatchSnapshot();
  });
});

