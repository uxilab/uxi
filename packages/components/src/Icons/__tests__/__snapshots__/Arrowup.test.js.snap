// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Arrowup /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 16 9.3"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <polyline
          points="8 0 0 8 1.3 9.3 8 2.6 14.7 9.3 16 8"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
